src/databricks/labs/remorph/helpers/recon_config_utils.py

```diff
@@ -1,10 +1,20 @@
+import json
 import logging
+import webbrowser
 
+from databricks.connect import DatabricksSession
+from databricks.labs.blueprint.installation import Installation
 from databricks.labs.blueprint.tui import Prompts
+from databricks.labs.blueprint.wheels import ProductInfo
+from databricks.labs.remorph.__about__ import __version__
+from databricks.labs.remorph.config import TableRecon
+from databricks.labs.remorph.reconcile.connectors.client import get_data_source
 from databricks.labs.remorph.reconcile.constants import SourceType
 from databricks.sdk import WorkspaceClient
 from databricks.sdk.errors.platform import ResourceDoesNotExist
 
+PRODUCT_INFO = ProductInfo(__file__)
```

Comment:
Why do you needs this? `WorkspaceClient` should already have this variable.

Reply:
@vijaypavann-db is this fixed?

Reply:
Done Sundar,  Hardcoded `product as remorph` for now, as `WorkspaceClient`  `product` variable is [not exposed](https://databricks-sdk-py.readthedocs.io/en/latest/clients/workspace.html)